File: parsers/dotnet/Taml.NET/TamlValue.cs
namespace TAML
{
	public class TamlValue
	{
		private string value;

		protected TamlValue() { }

		public TamlValue(string value)
		{
			this.value = value;
		}

		public override string ToString()
		{
			return value;
		}

		public static implicit operator string(TamlValue value) {

			return value.ToString();

		}

	}

}

File: src/JVM/taml4j-parser/src/main/java/com/taml4j/TamlValue.java
package com.taml4j;

/**
 * @author Luca Taddeo - Lucalas
 */
public class TamlValue {
	protected final String _value;

	public TamlValue() {
		this("");
	}

	public TamlValue(String value) {
		this._value = value;
	}

	@Override
	public String toString() {
		return _value;
	}


}
